Classes/Command/RunTestsCommand.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of TYPO3 CMS-based extension "b13/make" by b13.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ */
+
+namespace B13\Make\Command;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+use TYPO3\CMS\Core\Http\Client\GuzzleClientFactory;
+use TYPO3\CMS\Core\Package\PackageInterface;
+use TYPO3\CMS\Core\Service\MarkerBasedTemplateService;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * Command for creating docker based test environment for a TYPO3 extension
+ */
+class RunTestsCommand extends AbstractCommand
+{
+    /**
+     * @var SymfonyStyle $io
+     */
+    protected $io;
+
+    /**
+     * @var Filesystem $filesystem
+     */
+    protected $filesystem;
+
+    /**
+     * @var PackageInterface $package
+     */
+    protected $package;
+
+    protected function configure(): void
+    {
+        $this->setDescription('Setup runTests.sh to run tests, linter, cgl in a Docker environment');
+        $this->filesystem = GeneralUtility::makeInstance(Filesystem::class);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->io = new SymfonyStyle($input, $output);
+        $this->package = $this->askForPackage($this->io);
+
+        $templatesToCreate = [
+            [
+                'source' => 'https://raw.githubusercontent.com/TYPO3/styleguide/main/Build/Scripts/runTests.sh',
```

Comment:
Should we pin this to a specific commit, e.g. https://raw.githubusercontent.com/TYPO3/styleguide/936d5c5831fe1c0127460351f86204d9c03930c0/Build/Scripts/runTests.sh ?

Reply:
Hmm, my intention was to have a up-to-date script on each run. So we have changes automatically and do not have to maintain the template. If we pin it we could simply use a template (`Resources/Private/CodeTemplates/`) instead

Reply:
Using local templates now